lib/Exception/ModelNotFoundException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * @author 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
```

Comment:
```suggestion
 * @copyright 2024 Victor Lap <victorlap@outlook.com>
 *
 * @author 2024 Victor Lap <victorlap@outlook.com>
```

Or something similar. Please attribute the code to the people that actually wrote it.

Reply:
Alright, will do!